lib/mailgun.rb

```diff
@@ -20,7 +20,14 @@ module Mailgun
   def send_message(**params)
     make_valid!(params)
     text = params[:text]
+    url = params[:url]
     params[:text] = Social.email_template.get binding
+    email_notification = Social.email_notifications.to_h
```

Comment:
Instead of iterating through email_notifications hash, can we just check if there is a url passed in, and use the unsubscribe template instead?

Also, can we name the param `unsubscribe_url` instead?

Reply:
100% agree. Why my brain didn't think about it